packages/graph-explorer/src/modules/ConnectionDetail/ConnectionDetail.tsx

```diff
@@ -37,6 +37,12 @@ export type ConnectionDetailProps = {
 };
 
 const HEADER_ACTIONS = (isSync: boolean, isFileBased: boolean) => [
+  {
+    label: "Synchronize Database",
+    icon: <SyncIcon />,
```

Comment:
It would be nice to keep the loading state. `icon={<SyncIcon className={isSync ? "animate-spin" : ""} />}` should do the work.

Reply:
sure not a problem, looks so much better with the animation!